### DISCUSSION TITLE
Feature Request: Multi-language options

### DISCUSSION THREAD

**Discussion by p1ao12138** [ananthakumaran/paisa#264] [Ideas]
Can you add an option to select the language?

**Comment by punkyard** [comment]
i'm pushing this request: is Paisa available in other languages? is a project? do you need help with translations? I can do the French one if needed.
Great work by the way. With Svelte ✌️

**Comment by punkyard** [comment]
hi
best wishes for this new year

could we have a tiny bit of attention over here? ^^

is multi-language part of the Paisa roadmap?
is it possible to help in any way?
